toolchains/solidity/core/crates/slither-server/Cargo.toml

```diff
@@ -17,3 +17,7 @@ tokio = { version="1.34.0", features = ["full"] }
 tower-lsp = "0.20.0"
 colored = "2.0.4"
 thiserror = "1.0.50"
+glob = "0.3.1"
+toml = "0.8.8"
+tokio-utils = "0.1.2"
```

Comment:
tokio-utils (note the 's') is not used, please remove it from the package

Reply:
done